main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	"github.com/apache/beam/sdks/v2/go/pkg/beam"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/x/beamx"
+)
+
+var (
+	input_text = flag.String("input-text", "Default input text", "Input text to print.")
+)
+
+type Result = struct {
+	Pipeline    *beam.Pipeline
+	Scope       beam.Scope
+	PCollection beam.PCollection
+}
+
+func my_pipeline(input_text string) Result {
+	beam.Init()
```

Comment:
This should be called in `main`, right after flag parsing.

Reply:
Sounds good, moved.